res/css/views/settings/_UserProfileSettings.pcss

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
```

Comment:
Hello time travelling Dave

Reply:
Well, it's just a copy of a file with no (significant) changes so the copyright should surely remain? I guess I'll add 2024 though.